src/connectivity-api/connectivity-qt/connectivityqt/connectivity.cpp

```diff
@@ -387,6 +390,34 @@ void Connectivity::setHotspotEnabled(bool enabled)
 
     // TODO Remove this when SDK switch widget isn't broken
     d->m_propertyCache->setProperty("HotspotEnabled", enabled);
+
+    // HACK: For switching MediaTek /dev/wmtWifi adapter mode between AP/STA
+    //       when hotspot enabled state is changed.
+    try
```

Comment:
I dont see the need for try here, both write and open returns errors, none of them will trow afaik. 

https://doc.qt.io/qt-5/qiodevice.html#write
https://doc.qt.io/qt-5/qfile.html#open